### PR TITLE
add scored_suggest

### PR DESCRIPTION
**Pull request by kwhumphreys**
haven't used C for a while and haven't used C from Python before, but this seems to work

### CODE REVIEW

File: pyaspell.py [hunk 1]
```diff
@@ -139,6 +139,22 @@ def suggest(self, word):
 			raise TypeError("String expected")
 
 
+	def scored_suggest(self, word):
+		"""
+		Return list of spelling suggestions of given word.
+		Works even if word is correct.
+		"""
+		if type(word) is str:
```
Comment (cwharland) [inline pyaspell.py:147]: nit: PEP wants us to `isinstance(word, str)`

File: pyaspell.py [hunk 2]
```diff
@@ -139,6 +139,22 @@ def suggest(self, word):
 			raise TypeError("String expected")
 
 
+	def scored_suggest(self, word):
```
Comment (cwharland) [inline pyaspell.py:142]: Who is the 8 space monster!?!?!??!?!

File: pyaspell/pyaspell.py [hunk 3]
```diff
@@ -139,6 +139,22 @@ def suggest(self, word):
 			raise TypeError("String expected")
 
 
+	def scored_suggest(self, word):
+		"""
+		Return list of spelling suggestions of given word.
+		Works even if word is correct.
+		"""
+		if type(word) is str:
```
Comment (cwharland) [inline pyaspell/pyaspell.py:147]: Same `isinstance` here

**Review by cwharland — APPROVED** [review]
My C is super rusty but it looks fine. The duplication in the C code and the python wrapper is a bit of a mystery to me. Why do we need near identical doubles of each class/method?

**Comment by kwhumphreys** [comment]
duplication is from separate sets of files for Python 2 and Python 3

**Pull request merged**